pkg/etcd/etcd.go

```diff
@@ -52,8 +53,11 @@ type ClientInfo struct {
 	TrustedCAFile string
 }
 
-func (s *Server) Run(ctx context.Context, peerPort, clientPort string) (ClientInfo, error) {
+func (s *Server) Run(ctx context.Context, peerPort, clientPort string, walSizeBytes int64) (ClientInfo, error) {
```

Comment:
We're starting to get to the point where I think I'd like to see us consider either functional server options, or a config struct. Doesn't have to be in this PR though.